src/instance.ts

```diff
@@ -849,12 +913,9 @@ Please use the format 'my-instance' or '${bigtable.projectName}/instances/my-ins
    * Get Table objects for all the tables in your Cloud Bigtable instance.
    *
    * @param {object} [options] Query object.
-   * @param {boolean} [options.autoPaginate=true] Have pagination handled
```

Comment:
Sorry if I have overlooked this before, but is `autoPaginate` not a top-level option anymore? Does the user have to provide this through `options.gaxOptions.autoPaginate`? I think we should allow `options.autoPaginate`, in the same way we special-case `options.pageSize`,

Reply:
I followed example that was done in the Spanner repo https://github.com/googleapis/nodejs-spanner/pull/855#discussion_r408872152

WDYT?

Reply:
That was a pretty big PR to review, and it looks like I had too many streams of conversations to figure out how they all pieced together :\. In that case, did we get rid of "options.autoPaginate", but keep "options.pageSize"? That simply doesn't make sense, as they are both related options, and in order to give meaning to "pageSize", setting "autoPaginate" to "false" is required. Fortunately, we should be able to add that property back in to Spanner relatively easily, or alternatively, remove all other top-level pagination options.

Reply:
So for here let's go with the top level `autoPaginate` option and have API handle it the same way as `pageSize` and `pageToken`

Reply:
Cool, sounds good to me. Thank you!

Reply:
Added top level `autoPaginate` to `getAppProfiles` and `getTables`.